src/main/java/th/ac/kmitl/science/comsci/example/models/Profile.java

```diff
@@ -0,0 +1,9 @@
+package th.ac.kmitl.science.comsci.example.models;
+
+public abstract class Profile {
+    
+    private String ID;
```

Comment:
in Java variables naming convention, variables name should be writing in Lower Camal Case.
```java
ID -> id
Name -> name
```